middleware/middleware-1.3.19.md

```diff
@@ -0,0 +1,93 @@
+
+This issue mostly occurred when installing the Middleware as a service with limited permissions to system files. The issue and it's underlying cause have been resolved.
+
+## Bug fix: DSFinV-K threw exception when PayItem.Quantity was set to 0
+An issue was fixed that prevented the generation of DSFinV-K exports when the _Quantity_ of send _PayItem_ was set to 0. Although this issue was occurring very rarely, it has now been resolved.
+
+## Bug fix: DSFinV-K cash payment table included wrong data
+We altered the behavior of our DSFinV-K export to exclude non-cash payments in the `cash_per_currency.csv` file.
+
+## Bug fix: Fixed wrong time format in DSFinV-K transaction table
+We've updated our DSFinV-K export to correctly format the TSE's _start_ and _finish transaction_ timestamps in the `transactions_tse.csv`. Previously, these values were lacking the required milliseconds.
+
+Please note that as of now, all TSEs use unix timestamps, which only contain seconds (and no milliseconds). The last part of these values therefore is always '.000'.
+
+## Bug fix: Mono service could not be installed on some Linux versions
+Due to an issue in our Launcher, it was not possible to install the fiskaltrust.Middleware as a Mono service on some Debian-based Linux versions. This issue has been resolved.
+
+**Please note**: This change requires users to update the Launcher, i.e. to re-download the package that contains the `fiskaltrust.exe` from the Portal after rebuilding the Cashbox.
+
+## Affected packages
+Packages not listed here were not updated, as we decided to not increase the version of unchanged packages. All packages with versions greater or equal to 1.3.1 are compatible with each other (it is e.g. possible to use _fiskaltrust.Middleware.SCU.Swissbit.1.3.1_ with the new queue packages).
+
+- _fiskaltrust.Middleware.Queue.EF v1.3.19-rc1 *_
+- _fiskaltrust.Middleware.Queue.MySQL v1.3.19-rc1 *_
+- _fiskaltrust.Middleware.Queue.SQLite v1.3.19-rc1 *_
+- _fiskaltrust.Middleware.SCU.DE.CryptoVision v1.3.19_
+- _fiskaltrust.Middleware.SCU.DE.DieboldNixdorf v1.3.19_
+- _fiskaltrust.Middleware.SCU.DE.Epson v1.3.19_
+- _fiskaltrust.Middleware.SCU.DE.Swissbit v1.3.19_
+- _fiskaltrust.Launcher v1.3.19 **_
+
+\* An RC version was released to be able to deliver the bug fixes quickly. After verifying the correct behavior together with our partners, the versions will be re-labeled to non-RC.
+
+** Unlike regular package updates, this change requires a re-download of the Launcher from the Portal. The fiskaltrust.Launcher is currently not able to update itself.
+
+## Next steps in the Middleware
+We will continue to improve the stability of our Middleware in the next sprints. As always, we're happy to hear feedback and suggestions via [info@fiskaltrust.at](mailto:info@fiskaltrust.at) or directly via issues in our GitHub repositories.
```

Comment:
should be ".de" i guess

Reply:
We've always had .at here, as the same goes for the Portal release notes and technically both are wrong 😄 
I hope that we'll have a general email address for this stuff soon.